rollbar/__init__.py

```diff
@@ -1007,6 +1014,34 @@ def _build_bottle_request_data(request):
 
     return request_data
 
+
+def _build_wsgi_request_data(request):
+    request_data = {
+        'url': wsgiref.util.request_uri(request),
+        'user_ip': _wsgi_extract_user_ip(request),
+        'method': request.get('REQUEST_METHOD'),
+    }
+    if 'QUERY_STRING' in request:
+        request_data['GET'] = urlparse.parse_qs(request['QUERY_STRING'], keep_blank_values=True)
+        # Collapse single item arrays
+        request_data['GET'] = dict((k, v[0] if len(v) == 1 else v) for k, v in request_data['GET'].items())
+
+    request_data['headers'] = _extract_wsgi_headers(request.items())
+
+    try:
+        length = int(request.get('CONTENT_LENGTH', 0))
+    except ValueError:
+        length = 0
+    input = request.get('wsgi.input')
+    if length and input and input.seekable():
```

Comment:
I'm getting the following Exception while POSTing to a simple wsgi server:

`AttributeError: '_fileobject' object has no attribute 'seekable'`